api/api.go

```diff
@@ -107,7 +100,10 @@ func (d *DrivrAPI) FetchCertificateAuthority(ctx context.Context, issuer string)
 		logrus.WithError(err).Error("Failed to decode CA certificate")
 		return nil, err
 	}
-	return ca, nil
+
+	derBlock, _ := pem.Decode(ca)
```

Comment:
What is the meaning of "der"?

Reply:
DER (Distinguished Encoding Rules)
That is the format the certificate is in at that point. Actually PEM is the base64 encoded certificate in DER (which is binary) plus a header and a footer.

Reply:
Deshalb I would add a comment about DER :)